countlyCommon/countlyCommon/CountlyBase.cs

```diff
@@ -1038,7 +1053,12 @@ internal bool IsConsentGiven(ConsentFeatures feature)
 
         public async Task SetConsent(Dictionary<ConsentFeatures, bool> consentChanges)
         {
-            UtilityHelper.CountlyLogging("[CountlyBase] Calling 'SetConsent'");
```

Comment:
It's an error to remove this log call as that removes an important piece of information in the calling chain logs